src/web.security/jwt.encryption.ts

```diff
@@ -0,0 +1,42 @@
+import * as crypto from "crypto";
```

Comment:
If the 'web.security' package does remain, does this name follow/violate a naming standard? Just curious as I haven't seen any other packages with .s in.